pkg/scheduler/extender_test.go

```diff
@@ -252,6 +257,35 @@ func TestGenericSchedulerWithExtenders(t *testing.T) {
 			},
 			name: "test 9",
 		},
+		{
+			registerPlugins: []st.RegisterPluginFunc{
+				st.RegisterFilterPlugin("TrueFilter", st.NewTrueFilterPlugin),
+				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
+				st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
+				st.RegisterPostFilterPlugin(defaultpreemption.Name, func(plArgs apiruntime.Object, fh framework.Handle) (framework.Plugin, error) {
```

Comment:
Hmm... the original idea was to verify the registered extenders are passed all the way to defaultpreemption plugin, and also **get called**.

But this UT only tests on `schedule.Schedule()`, which doesn't proceed for run PostFilter plugins, and hence cannot verify if the extenders are called or not:

https://github.com/kubernetes/kubernetes/blob/9ff3b7e744b34c099c1405d9add192adbef0b6b1/pkg/scheduler/scheduler.go#L540

We may pursue it in some test (e.g., pkg/scheduler/scheduler_test.go) that covers `scheduleOne()`, or directly test `RunPostFilterPlugins()`. (the previous option works better)